salt/iiif/loris.sls

```diff
@@ -42,13 +44,16 @@ loris-dependencies:
             - libfreetype6
             - libfreetype6-dev
             - zlib1g-dev
+            {% if trusty %}
```

Comment:
no idea on what the effect will be, but we can recreate `iiif--ci` once ready to test the whole corpus of images

Reply:
yes, I was going to mention this. it builds on vagrant but has a few failed states on aws due to no newrelic config. I don't think upstart ever got that far because of the duplicated line

Reply:
https://ci--alfred.elifesciences.org/job/process-iiif/ usually deploys this, seemed green until ~15 days ago. Maybe it has problems on a fresh server?